src/components/Overlay/README.md

```diff
@@ -0,0 +1,32 @@
+<!--GITHUB_BLOCK-->
+
+# Loader
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {Overlay} from '@gravity-ui/uikit';
+```
+
+The `Overlay` component renders an overlay over the parent element with relative position.
+For example, it can be used to preserve the desired layout while loading data.
+
+### PropTypes
+
+| Name      | Type               | Required | Default | Description                         |
+| :-------- | :----------------- | :------: | :------ | :---------------------------------- |
+| className | `String`           |          |         | CSS class name of the root element  |
+| visible   | `Boolean`          |          | `false` | Overlay visibility state            |
+| view      | `"base"` `"float"` |          | `base`  | Overlay background style            |
+| children  | `React.ReactNode`  |          |         | Content, usually a Loader component |
+
+### Basic usage
+
+```jsx
+<div style={{position: 'relative'}}>
+  <div>Some content to hide under overlay</div>
+  <Overlay visible={loading}>
+    <Loader />
```

Comment:
@amje @korvin89 are you sure that this use case is really convenient? Do I transfer the same component to the loader every time? It looks like 90% of all uses here will be such a `<Loader />` component and it should be made default by default

Reply:
Yes, I believe it's better to be that way, Overlay is a utility component, it shouldn't have "default" look. The look will depend on the usage context. Also, turning off the default behaviour would look a bit wierd:
```tsx
<Overlay visible children={null} />
```